pyspark-iris/{{ cookiecutter.repo_name }}/.ipython/profile_default/startup/00-kedro-init.py

```diff
@@ -52,6 +53,7 @@ def reload_kedro(path, line=None, env: str = None, extra_params: Dict[str, Any]
         for module in to_remove:
             del sys.modules[module]
 
+        configure_project(metadata.package_name)
```

Comment:
Nice thank you, these definitely need to go in the `0.17.1` release of the starters.